docs/reference/getting-started.md

```diff
@@ -0,0 +1,110 @@
+2. In OpenTelemetry, _service_ means _an entity that produces telemetry_, so this is where your application name should go. For more information, go to [FAQs](faq.md#why-service).
+3. This is the Elastic endpoint where all your telemetry will be exported. If you don't have one yet, check out [how to get it](how-tos.md#get-export-endpoint).
+4. Use an API key to connect the agent to the {{stack}}. If you don't have an API key yet, check out [how to create one](how-tos.md#create-api-key).
+
+:::{include} _snippets/tip-provide-values-from-outside.md
+:::
+
+## Hello World!
+
+The agent is fully initialized, so now you can start sending telemetry to your {{stack}}! Here's a quick example of manually creating a [span](https://opentelemetry.io/docs/concepts/signals/traces/#spans) and finding it in {{kib}}:
+
+### Generate telemetry
+
+```kotlin
+val agent = ElasticApmAgent.builder(application)
+    .setServiceName("My app name")
+    //...
+    .build()
+
+
+agent.span("My Span") {
+    Thread.sleep(500) // <1>
+    agent.span("My nested Span") { // <2>
+        Thread.sleep(500) 
+    }
+}
+```
+1. This is to simulate some code execution for which we want to measure the time it takes to complete.
+2. This is to demonstrate how span hierarchies look like in {{kib}}.
+
+### Visualize telemetry
+
+Once your app has sent telemetry data, either [manually](manual-instrumentation.md) or [automatically](automatic-instrumentation.md), you should be able to visualize it in {{kib}} by navigating to **Applications -> Service Inventory** in the main menu, or alternatively, searching for "Service Inventory" in the [global search field](https://www.elastic.co/guide/en/kibana/current/introduction.html#kibana-navigation-search).
+
+You should find your application listed there.
+
+:::{image} ../images/span-visualization/1.png
+:screenshot:
+:width: 350px
+:::
+
+When you open it, go to the **Transactions** tab, where you should see your app's "outermost" spans listed.
+
+:::{image} ../images/span-visualization/2.png
+:screenshot:
+:width: 350px
+:::
+
+And after clicking on the span, you should see it in detail.
+
+:::{image} ../images/span-visualization/3.png
+:screenshot:
+:::
+
+## What’s next? [whats-next]
+
+- This guide uses the minimum configuration options needed to initialize the agent. If you'd like to explore what else you can customize, take a look at the [configuration page](configuration.md).
+
+- In our [simple example](#hello-world), we've _manually_ sent a span, so we essentially created some [manual instrumentation](manual-instrumentation.md) for our app. That's quite helpful (and flexible), however, the agent can create _automatic instrumentations_ as well! Meaning that, only by initializing the agent, it will start sending telemetry data on your behalf without you having to write code. For more details, refer to [Automatic instrumentation](automatic-instrumentation.md).
```

Comment:
```suggestion
- In the [simple example](#hello-world), you've _manually_ sent a span, so you've created some [manual instrumentation](manual-instrumentation.md) for your app. While helpful and flexible, the agent can also create _automatic instrumentations_. This means that by simply initializing the agent, it will start sending telemetry data on your behalf without you having to write code. For more details, refer to [Automatic instrumentation](automatic-instrumentation.md).
```